README.md

```diff
@@ -1,45 +1,59 @@
-# vackup-docker-extension
-
-[![Build, Scan and Push](https://github.com/felipecruz91/vackup-docker-extension/actions/workflows/build-scan-push.yaml/badge.svg)](https://github.com/felipecruz91/vackup-docker-extension/actions/workflows/build-scan-push.yaml)
-[![Lint Dockerfile](https://github.com/felipecruz91/vackup-docker-extension/actions/workflows/hadolint.yaml/badge.svg)](https://github.com/felipecruz91/vackup-docker-extension/actions/workflows/hadolint.yaml)
-
-This repository contains a Docker Extension that backups Docker volumes. This extension is purely a demonstration of how it'd be to convert the [Vackup](https://github.com/BretFisher/docker-vackup) project by Bret Fisher into a Docker Extension.
-This example is part of the YouTube Livestream [Introducing Docker Extensions](https://www.youtube.com/watch?v=BHKp7Sc3VVc) hosted by Bret Fisher.
-
-![Extension Screenshot](./docs/images/screenshot-1.png)
-
+# Volumes Backup and Share Extension
+
+[![Build, Scan and Push](https://github.com/docker/vackup-docker-extension/actions/workflows/build-scan-push.yaml/badge.svg)](https://github.com/docker/vackup-docker-extension/actions/workflows/build-scan-push.yaml)
+[![Lint Dockerfile](https://github.com/docker/vackup-docker-extension/actions/workflows/hadolint.yaml/badge.svg)](https://github.com/docker/vackup-docker-extension/actions/workflows/hadolint.yaml)
+
+
+![Extension Screenshot](./docs/images/1-table.png)
+
+🚀 This extension was originally created by [Felipe Cruz](https://github.com/felipecruz91/)
+
+## Features
+- Export a volume:
+  - To a compressed file in your local filesystem
+  - To an existing local image
+  - To a new local image
+- To a new image in Docker Hub (or another registry)
+- Import data into a new container or into an existing container:
+  - From a compressed file in your local filesystem
+  - From an existing image
+  - From an existing image in Docker Hub (or another registry)
+- Transfer a volume via SSH to another host that runs Docker Desktop or Docker engine.
+- Clone a volume
+- Empty a volume
+- Delete a volume
 ## Installation
 
 Install Docker Extension with the [Extensions CLI](https://docs.docker.com/desktop/extensions-sdk/#prerequisites).
 
 ```bash
-  docker extension install felipecruz/vackup-docker-extension:latest
+  docker extension install docker/volumes-backup-extension:latest
 ```
 
 ## Run Locally
 
 Clone the project
 
 ```bash
-  git clone https://github.com/felipecruz91/vackup-docker-extension.git
+  git clone https://github.com/docker/vackup-docker-extension.git
```

Comment:
GH takes care of the repo rename redirects when cloning, but nonetheless, I think we should rename it here:

```suggestion
  git clone https://github.com/docker/volumes-backup-extension.git
```